README.md

```diff
@@ -1,2 +1,83 @@
 # in-toto
 in-toto is a series of scripts to protect the supply chain integrity.
+
+# About in-toto
+in-toto guarantees that the end-user (or client) is able to verify that the entire development life cycle has been conducted as per the specified layout and that each of the functionaries (eg. developers) have performed the specified tasks and there haven't been any malicious changes in the files.
+
+in-toto requires a project layout that specifies the functionaries and the tasks they are supposed to perform.
+After each functionary performs its task, a link metadata is generated.
+This metadata is used to verify the intermediate and final products with the project layout.
+
+# in-toto demo
+You can try in-toto by running the demo application at [demo application](https://github.com/in-toto/in-toto/tree/develop/demo).
+The demo basically outlines three users viz., Alice (project owner), Bob (functionary) and Carl (functionary) and how in-toto helps to specify a project layout and verify that the layout has been followed in a correct manner.
+
+# Getting Started
+## 1. Installation
+
+*Make sure you have git, python and pip installed on your system*
+*and get in-toto*
+
+git clone -b develop --recursive https://github.com/in-toto/in-toto.git
```

Comment:
I would:

1.- put this in a code block (you can use  \`\`\` to start and end a code block
2.- Now that the develop branch is the default branch, we don't need the `-b develop` switch. We also need to correct this in the demo text.